modules/leaderboard/utils/types.ts

```diff
@@ -0,0 +1,40 @@
+import {
+    Leaderboard as PrismaLeaderboard,
+    Entry as PrismaLeaderboardEntry,
+} from "../_gen/prisma/index.d.ts";
+
+export interface Leaderboard {
+    key: string;
+    name: string;
+
+    locked: boolean;
+    createdAt: string;
+    updatedAt: string;
+}
+
+export interface LeaderboardEntry {
+    leaderboardKey: string;
+    ownerId: string;
+    score: number;
+
+    createdAt: string;
+    updatedAt: string;
+}
+
+export type SortType = "hi_is_better" | "lo_is_better";
```

Comment:
Nit: these are usually called "asc" and "desc"

Reply:
When you say that, do you mean that `asc` means "higher score is better," or do you mean the opposite?

Reply:
Another way of phrasing it:
Does ascending mean "lowest score is 1st" or "highest score is 1st"?

Reply:
List order is typically from the start to the end of the list, so ascending means it starts at the lowest number and goes up

Reply:
Sounds good!